tests/test.validator.js

```diff
@@ -85,6 +85,18 @@ describe('Validator', function() {
     assert.equal(parsedJSON.summary.warnings, 0);
   });
 
+  it('should send JSScanner messages to the collector', (done) => {
```

Comment:
Fair enough. The test XPI is one line of JS but who knows what other tests it will trip once we have more.

Just checking that the collector increments at all is what I'm looking for, so how about I just check for `> 0`? That aspect of the code isn't tested anywhere else and I figure why not leave it in?